### DISCUSSION TITLE
what about a go gopher logo for ddns-updater

### DISCUSSION THREAD

**Discussion by fredericrous** [qdm12/ddns-updater#156] [Show and tell]
Hey, I'm not a designer but I love making logos. The current logo of ddns-updater is a matchup of a bunch of logos of providers, plus docker. It's not very personal, isn't it ^^

What about the following one. Let me introduce DDNS Gopher:

![image](https://user-images.githubusercontent.com/702227/106057568-0f482e80-60e8-11eb-94af-2fa735680d55.png)

I wanted to represent the Dynamic of Dynamic-DNS and got the idea of a lightning bolt, then I thought of an electrocuted gopher and tried few things

Below is my process creation with the different ideas I had

![image](https://user-images.githubusercontent.com/702227/106058366-0c9a0900-60e9-11eb-817b-504c1dec7236.png)

What do you think? If you like it, I'll open a PR and will version the source file

**Comment by qdm12** [comment]
Awesome thanks!

Yes i definitely wanted to change the logo for a while.

Maybe make it as an SVG so it can easily be remodeled and is also tiny size wise?

I use [that Gopher SVG as a default template](https://github.com/qdm12/go-template/blob/main/title.svg) and you can modify it with Inkscape for example. Feel free to change its color from blue to green/pink/whatever too!

I like the ⚡  and DNS appearing somewhere on it 👍 the server icon is a nice touch too.

**Reply by fredericrous** [reply]
I'm glad you like it ^^
I started from the following template to make the logo https://github.com/golang-samples/gopher-vector/blob/master/gopher.ai
I opened PR https://github.com/qdm12/ddns-updater/pull/157 with the svg and readme updated. I followed your suggestion and changed the color to green 😄

I also tried to make a logo where we see the skeleton

![image](https://user-images.githubusercontent.com/702227/106076027-a91ed400-6106-11eb-9b75-2dae735497f3.png)

This one is fun but might be too violent xD I think I prefer the first one

**Reply by qdm12** [reply]
Haha that's pretty cool!! Although it did took me a few seconds to get what the skeleton was shown 😄 Just the mouth seems to be under its backbone 😄 I think we can use it as well, up to you though, I'm happy with any of the two.

**Reply by fredericrous** [reply]
I reworked the skeleton a bit just for fun
![image](https://user-images.githubusercontent.com/702227/106156043-f9cf1500-6178-11eb-81b0-890598bc3cc4.png)

but anyway, unless we want an animated logo, maybe let's stick with the one we have now. Also this might inspire someone with better logo creation skills :P 

I created a PR for a favicon https://github.com/qdm12/ddns-updater/pull/159

![image](https://user-images.githubusercontent.com/702227/106158245-74009900-617b-11eb-9619-609bb7591288.png)

**Comment by AdyGCode** [comment]
My version of a loGO reduces it to an arrow and DDNS.

![ddns-updater@0 5x](https://github.com/qdm12/ddns-updater/assets/60425995/5d7499be-095a-492a-80ef-c13f8ffe2e17)

I'll look at a GOphe-based one over the next few weeks...

**Comment by AdyGCode** [comment]
OK, as promised - a Gopher version of the loGOpher ... 

![ddns-updater-favicon@64w](https://github.com/qdm12/ddns-updater/assets/60425995/bafc066e-6519-400b-b8f1-ad64ed52081e)

![ddns-updater-favicon@1 0x](https://github.com/qdm12/ddns-updater/assets/60425995/05e4b80b-4fbe-40bd-b125-87fbdaa697df)

![ddns-updater@64w](https://github.com/qdm12/ddns-updater/assets/60425995/62ec0515-dedd-4a4a-8a8e-e37558283250)

![ddns-updater@1 0x](https://github.com/qdm12/ddns-updater/assets/60425995/af1eadad-b3c8-4833-9c56-7c58a345b759)
